composer.json

```diff
@@ -5,19 +5,20 @@
     "license": "MIT",
     "type": "project",
     "require": {
-        "php": ">=7.2.0",
-        "laravel/framework": "^6.0",
-        "twilio/sdk": "^5.0"
+        "php": ">=8.1.0",
+        "laravel/framework": "^10.0",
+        "twilio/sdk": "^5.0",
```

Comment:
I think it may be worth updating the SDK at some point, too: https://github.com/twilio/twilio-php/releases/tag/7.16.1 Can you please create a ticket for that?

Reply:
@CristhianMotoche sure thing!

Reply:
Here's the ticket https://twilio-productivity.atlassian.net/browse/DEVEDSB-6805